docs/program/run.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Run SDK Code"
+linkTitle: "Run Code"
+weight: 50
+type: "docs"
+description: "Execute the logic you've written to control your robot or fleet."
+images: ["/services/img/icons/sdk.svg"]
+tags: ["client", "sdk", "application"]
+---
+
+After saving your [code sample](/program/#hello-world-the-code-sample-tab) and adding control logic with [Viam's SDKs](/program/sdks/), run your program to control your Viam-connected robot.
+Do this by running a command to execute the program in the terminal of a machine with the appropriate programming language and Viam SDK installed.
+
+For example:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go run ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For information on running C++ code see [the instructions on GitHub](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you are connecting to your robot through an external signaling address, as is the case in the boilerplate code samples generated by the Viam app, you can perform this step on any computer.
+As long as this computer is able to establish a network connection with the robot's computer, the control logic will be executed on the robot.
```

Comment:
```suggestion
This is useful because as long as this computer is able to establish a network connection with the robot's computer, the control logic will be executed on the robot.
```